### ISSUE TITLE
Question about metric.

### ISSUE THREAD

**Issue by GuHuangAI** [ktertikas/part_nerf#10]
How to calculate the metric? 
COV and MMD seems to be aplied in point cloud data.

**Comment by ktertikas** [comment]
Thanks for the interest in our work! 

We calculate the COV and MMD metrics using points sampled from the surface of our generated shapes. Specifically, we randomly generate 1000 shapes for each Shapenet category, and sample 2048 points from each shape. Then, we compare to 500 shapes from the training set and 500 shapes from the test set (points are sampled for these shapes as well).

Please have a look at section A.5 of the supplementary material of the paper for more details, and let me know if you have any other questions!

**Comment by ktertikas** [comment]
Closing the issue, feel free to reopen if there are any other questions!

**Comment by GuHuangAI** [comment]
@ktertikas Hello, could you please release the code for calculating metrics?
